modules/tinymce/src/themes/silver/test/ts/atomic/ui/core/complex/utils/SelectTest.ts

```diff
@@ -0,0 +1,47 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import { Delimiter, makeBasicSelectItems } from 'tinymce/themes/silver/ui/core/complex/utils/Select';
+
+describe('atomic.tinymce.themes.silver.ui.core.complex.utils.Select', () => {
+  context('makeBasicSelectItems', () => {
+    it('Works with a semicolon delimiter', () => {
+      const config = 'Paragraph=p; Heading 1=h1;Heading 2=h2; Heading 3 =h3;Pre =pre';
+      const items = makeBasicSelectItems(config, Delimiter.SemiColon);
+      assert.isArray(items);
+      assert.lengthOf(items, 5);
```

Comment:
Length doesn't need to be checked if you're doing a `deepEqual` - the length will be part of that check already.

It should also print a much nicer error message than "expected length of 5 was 6" or whatever